docs/FAQ/FAQ-Wallet.md

```diff
@@ -1,133 +1,77 @@
+The only wallets that allow configurable gap-limit value are [Electrum](/ElectrumWallet.md) and [Wasabi](/WasabiWallet.md)
 
-If the problem persists, contact the owner of the BTCPay Server: The reverse proxy of the BTCPay server hosting BTCPay might not support Websockets.
+If you'd like to use an external wallet to manage the funds, we recommend that perform a wallet recovery with those two wallets.
 
-To confirm this is the issue, create a new invoice and go on its checkout page, you can also go on your store's "Wallet" page.
-You should then see this error in the Javascript console:
+- [Increasing gap limit in Electrum](/ElectrumWallet.md/#configuring-the-gap-limit-in-electrum)
+- [Increasing gap limit in Wasabi](/WasabiWallet.md/#configuring-the-gap-limit-in-wasabi)
 
-```
-WebSocket connection to ‘wss://pay.example.com/i/4yhCmpWxJcHfVG3rV4EmEu/status/ws’ failed: Error during WebSocket handshake: Unexpected response code: 404
-```
+After you've increased the gap limit, the balance in your external wallet and btcpay wallet should match. If they don't you may have set up your derivation scheme incorrectly.
 
-To fix the situation, if your reverse proxy is nginx, make sure that the following is included at the top of `/etc/nginx/conf.d/default.conf` :
+#### 2. Use the internal wallet
 
-```
-# If we receive Upgrade, set Connection to "upgrade"; otherwise, delete any
-# Connection header that may have been passed to this server
-map $http_upgrade $proxy_connection {
-  default upgrade;
-  '' close;
-}
-proxy_set_header Upgrade $http_upgrade;
-proxy_set_header Connection $proxy_connection;
-```
-
-Then restart nginx.
-
-```
-/etc/init.d/nginx reload
-```
-## Receiving address in Zap and Electrum is different?
-
-Zap and Electrum are two different wallets. Zap is for [Lightning Network](LightningNetwork.md) and Electrum is for on-chain transactions. They are unrelated and use different private keys. Use Electrum or internal BTCPay wallet to check your on-chain payments and Zap for your Lightning Network payments.
-
-In the future, there will be wallets merging both on-chain and off-chain transactions into one, but for now, you have to use them separately.
-
-## Can I see my payments on mobile?
-
-If you wish to follow your on-chain payments on a mobile device, you can use a watch-only wallet like [Sentinel](https://play.google.com/store/apps/details?id=com.samourai.sentinel) for Android and [bitWallet](https://itunes.apple.com/us/app/bitwallet-bitcoin-wallet/id777634714) or [ArcBit](https://itunes.apple.com/ca/app/arcbit-bitcoin-wallet/id999487888) for iOS. Watch-only wallets allow xpub import and don't require the private key.
+For best user-experience and privacy, we recommend that you consider dropping external wallets and start using the [BTCPay Server's internal wallet](/Wallet.md).
```

Comment:
This sentence is out of context.
```suggestion
To automatically resolve the gap limit problem, we recommend that you consider dropping external wallets and start using the [BTCPay Server's internal wallet](/Wallet.md).
```

Reply:
It's not it touched upon privacy issue.

Reply:
But the initial question of that section is about the gap limit. It would make sense to me that this sentence answers the question in that context ?